content/docs/deployment/rpm/modules/upgrade/authorization.md

```diff
@@ -15,6 +15,7 @@ description: >
 This section outlines the upgrade steps for Container Storage Modules (CSM) for Authorization.  The upgrade of CSM for Authorization RPM is handled in 2 parts:
 - Upgrading the CSM for Authorization proxy server
 - Upgrading the Dell CSI drivers with CSM for Authorization enabled
+The Authorization proxy server should be upgraded before upgrading the CSI Driver and Authorization sidecar.
```

Comment:
Instead of "should", we should use "must"

Reply:
done